release-team/role-handbooks/enhancements/README.md

```diff
@@ -102,23 +109,38 @@ It is important that this process be followed and documentation remain up-to-dat
+    - Add the following github teams with **Admin** access
+      - @release-team-leads
+    - Add the following github teams with  **Write** access
+      - @production-readiness
+      - @release-team-comms
+      - @release-team-docs
+      - @release-team-enhancements
+  - Create all the Fields (columns) for the board
+    > Note: This is currently a manual process and  <https://github.com/orgs/community/discussions/41133> filed to help automate the process.
+    - Until this can be automated; manually create fields with the same `Field Name`, `Field type`, and `Option` values from the previous release's project board.
+  - Create all the Views for the board
+    > Note: This is also currently a manual process until github GraphQL APIs allow for manipulating Views
+    - Until this can be automated; manually create Views with the same Names and Fields from the previous release's project board.
+      Tip: You can easily view all the Fields present in each view by clicking the `v` next to the View's name an selecting `Configuration` -> `Fields`
+      Note: Remember to **save** each View. If there is a blue bubble next to the Views name there are unsaved changed for that view!
+- Create a shortlink for the Tracking Board
+  - Create a free account on [bitly](https://bitly.com/) to create a shortlink for the new Enhancement Tracking Board following the pattern `k8sxyy-enhancements` e.g., https://bit.ly/k8s127-enhancements.
+- Make a [pull request](https://github.com/kubernetes/sig-release/pull/1411) to add the shortlinked Enhancement Tracking Board to the current release page in [sig-release][sig-release].
+- Make a pull request to add the shortlinked Enhancement Tracking Board to [kubernetes/enhancements README page](https://github.com/kubernetes/enhancements#enhancements-tracking-board).
 - Find [Issues][enhancements-issues] from previous milestone that have graduated to Stable. Remove `tracked/yes` or `tracked/no` labels. Check to see if the KEP status has been updated to `implemented`. If it has, close the issue. If it has not, ask the issue contact to both update the KEP status field and close the Enhancement issue once the update PR has merged.
 - Find [Issues labeled `tracked/yes`](https://github.com/kubernetes/enhancements/issues?q=is%3Aopen+is%3Aissue+label%3Atracked%2Fyes) and change to `tracked/no` until the Enhancement is ready to be tracked for the upcoming release.
 - Close previous milestone by ensuring that there are no open issues/PRs in that milestone.
 - Gather Shadows to have them read this handbook and give expectations on what the process looks like and their particular role. If possible, try to schedule a call with the shadows to get them accustomed to the team. This helps as a great team building exercise.
-- Add Shadow information to the `Enhancements` tab of the Enhancements Tracking sheet.
+- Add Shaodws to the  `release-team` and `release-team-enhancements` GitHub teams. (See [access-required](#access-required))
```

Comment:
```suggestion
- Add Shadows to the `release-team` and `release-team-enhancements` GitHub teams. (See [access-required](#access-required))
```

Reply:
Hey @marosset, just a suggestion (copied from [Leo's addition to the Release Lead Handbook](https://github.com/kubernetes/sig-release/pull/2100/files#diff-176705076aa18b845e0fcb445f9cb129010094bb7ac6e983a5372703d8b8d594R76-R77)).

Maybe we can also add the following bit about the creation of `Enhancements shadow rotation sheet` (for enhancement role's updates in the weekly release team meetings) in Week 0/1 tasks.

```
 [ ] Shadow rotation sheet created for release team meeting enhancements role updates
  - Access: : **restricted access**, edit rights shared with release team enhancements shadows individually
```